kitsune/search/v2/signals/questions.py

```diff
@@ -0,0 +1,62 @@
+from django.db.models.signals import post_save, post_delete, m2m_changed
+from django.dispatch import receiver
+from django.conf import settings
+from kitsune.search.v2.es7_utils import (
+    index_object,
+    delete_object,
+    reindex_by_query,
+    index_objects_bulk,
+)
+from kitsune.questions.models import Question, QuestionVote, Answer, AnswerVote
+from taggit.models import Tag
+
+
+@receiver(post_save, sender=Question)
+@receiver(m2m_changed, sender=Question.tags.through)
+def handle_question_save(instance, **kwargs):
```

Comment:
docstrings pls

Reply:
I won't add comments to the rest of the signals but let's do this for all the signals

Reply:
IMO `handle_question_save` is pretty descriptive, what extra detail would you like to see in the docstrings?